src/components/ChartTitle.jsx

```diff
@@ -1,80 +1,78 @@
-import React, { Component } from 'react';
+import React from 'react';
 import ReactDOM from 'react-dom';
 import { connect } from '../store/Connect';
 import '../../sass/components/_chart-title.scss';
 
-class ChartTitle extends Component {
-    componentDidUpdate(prevProps) {
-        const { updateProps, ...props } = this.props;
-        const { updateProps: prevUpdateProps, ...previousProps } = prevProps;
-        if (previousProps.open_market !== props.open_market || previousProps.open !== props.open) {
-            updateProps(props);
-        }
-    }
-
-    render() {
-        const {
-            chartId,
-            ChartTitleMenu,
-            containerId,
-            currentSymbol,
-            enabled,
-            isMobile,
-            MarketSelector,
-            portalNodeId,
-            onChange,
-            SymbolSelectButton,
-            setMenuOpen,
-            searchInputClassName,
-            theme,
-            onMouseEnter,
-            onMouseLeave,
-            isNestedList,
-        } = this.props;
+const ChartTitle = props => {
+    const {
+        chartId,
```

Comment:
just per curiosity, why not replace this with `props` on line `const ChartTitle = props => {` ?